src/redownload/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class RedownloadException(Exception):
```

Comment:
It makes me so happy that you define exception classes. Usually when I do this I add a docstring that explains more about the meaning of an exception. Up to you.